libs/crossplane/config.jsonnet

```
@@ -88,9 +88,9 @@ config.new(
     // Upbound official providers
     // https://marketplace.upbound.io/
     {
-      output: 'upbound-provider-aws/0.40',
+      output: 'upbound-provider-aws/1.6.1',
```

Note that if we have changed the version in CRDs (line 93), we should update this one to match:

```suggestion
      output: 'upbound-provider-aws/1.6.0',
```